lib/Log/Any/Adapter/DERIV.pm

```diff
@@ -5,10 +5,11 @@ use strict;
 use warnings;
 
 # AUTHORITY
-our $VERSION = '0.006';
+our $VERSION = '0.007';
```

Comment:
we needn't decrease this number. that's dzil's job to manager this number

Reply:
ok I kept it to 0.007 as it changed to 0.008